packages/core/src/sdk/product/useProductGalleryQuery.ts

```diff
@@ -65,6 +65,65 @@ type ProductGalleryQueryOptions = {
   term: ClientManyProductsQueryQueryVariables['term']
 }
 
+type updateSearchParamsType = {
+  selectedFacets: Facet[]
+  updatedFuzzyFacetValue: string
+  updatedOperatorFacetValue: string
```

Comment:
question: could there be a problem if it comes null/undefined?

Reply:
I'll update these types, but I think it's not a problem since we checked the value more than once

like [here](https://github.com/vtex/faststore/blob/ce3696a9ce4a37171191e24f0baf78cbf09f3d00/packages/core/src/sdk/product/useProductGalleryQuery.ts#L98) and [here](https://github.com/vtex/faststore/blob/ce3696a9ce4a37171191e24f0baf78cbf09f3d00/packages/core/src/sdk/product/useProductGalleryQuery.ts#L116) (adding fallbacks)

Reply:
types updated [here](https://github.com/vtex/faststore/pull/2721/commits/7637d9089f964553eb5db23dcc4ab67ec8473cf8).